versions update

rnovikovP opened the pull request:
according to https://www.percona.com/services/policies/percona-software-support-lifecycle#pt

rnovikovP committed:
versions update 

according to https://www.percona.com/services/policies/percona-software-support-lifecycle#pt

denisok commented:
OK, and do we have those packages ?

denisok commented on docs/installation.rst:
```diff
@@ -6,14 +6,12 @@ Installing Percona Toolkit
-* Ubuntu 17.04 (Zesty Zapus)
-* Red Hat Enterprise Linux or CentOS 6 (Santiago)
-* Red Hat Enterprise Linux or CentOS 7 (Maipo)
+* Debian 8 ("Jessie")
```
https://www.percona.com/downloads/percona-toolkit/LATEST/  there is no Debian 8 choice...

rnovikovP committed:
correcting Debian versions (forgot to change them)

according to https://www.percona.com/services/policies/percona-software-support-lifecycle#pt
